scripts/protoc-swagger-gen.sh

```diff
@@ -19,7 +19,8 @@ done
 # combine swagger files
 # uses nodejs package `swagger-combine`.
 # all the individual swagger files need to be configured in `config.json` for merging
-swagger-combine ./client/grpc-gateway/config.json -o ./client/grpc-gateway/swagger.json --continueOnConflictingPaths true --includeDefinitions true
+swagger-combine ./client/docs/config.json -o ./client/docs/swagger-ui/swagger.yaml -f yaml --continueOnConflictingPaths true --includeDefinitions true
 
 # clean swagger files
-find ./ -name 'query.swagger.json' -exec rm {} \;
+rm -rf cosmos
+rm -rf ibc
```

Comment:
> Also- is it possible to mark certain API endpoints (e.g. all legacy REST endpoints) as deprecated/soon-to-be-deprecated in Swagger docs?
> 

Yes 100%, I'll address this in a followup PR

> Can we output these to a default root directory other than `./` ?
> 
> As it is now, this would cause problems if in the future we add a `./ibc` or `./cosmos` directory for any purpose besides these swagger auto generated files.

Correct, but we don't have custom output directory support for `protoc-sawgger-gen`. I have created an issue upstream to add support for that. For now,
- We can revert this to remove all generated files (`find ./ -name 'query.swagger.json' -exec rm {} \;`) first and then remove `./cosmos`, `./ibc` directories if they are empty. wdyt?

Reply:
We have a way to override the output directory for swagger generated files. It;s being addressed here: #7374